src/core/heating_cooling_loads.jl

```diff
@@ -202,17 +202,17 @@ end
         blended_doe_reference_percents::Array{<:Real,1} = Real[],
         annual_tonhour::Union{Real, Nothing} = nothing,
         monthly_tonhour::Array{<:Real,1} = Real[],
-        fuel_loads_ton_per_hour::Array{<:Real,1} = Real[],
+        thermal_loads_ton::Array{<:Real,1} = Real[],
```

Comment:
should we append `_per_time_step` to `thermal_loads_ton`? (or is it assumed to be per hour and so we append `_per_hour`?)

Reply:
@NLaws `ton` (refrigeration ton) is a unit of **power**, so it's equivalent to kW where we don't need the `_per_time_step`